torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -1,44 +1,100 @@
 defmodule Torchx.NxDoctestTest do
+  @moduledoc """
+
+  # Why some doctests are excluded?
+
+  ## Not callbacks
+
+  * default_backend/2
+
+  ## Python rounds differently, than Elixir (for now)
+
+  * round/1
+
+  ## Result mismatch in less significant digits (precision issue)
+
+  * logistic/1
+
+  ## Callback or test requires explicity backend transfer to BinaryBackend
+
+  * to_binary/2
+  * to_flat_list/2
+  * random_uniform/4
+  * random_normal/4
+  * atan2/2
+
+  ## Not all options are supported
+
+  * argmax/2
+  * argmin/2
+
+  ## Unsigned ints not supported by Torch
+
+  * quotient/2
+  * mean/2
+
+  ## Output mismatch
+
+  * qr/2
+  * dot/4
+  * cholesky/1
```

Comment:
We need to add tests for all operations that are missing doctests. So this means tests for: qr, dot, cholesky, quotient, mean, argmin, argmax, logistic, round, atan2, random_uniform and random_normal. :)